src/main/java/com/google/codeu/servlets/FormHandlerServlet.java

```diff
@@ -172,6 +172,11 @@ private void handleEatenFood(HttpServletRequest request, HttpServletResponse res
         if (blobKey == null) {
             out.println("<p>You ate " + meal.getAmount() + " of "  + meal.getFood() +
                     " on " + format.format(meal.getDate()) + ".</p>");
+            if (meal.getCO2() == 0){
+               out.println("This Food is currently not in our Database. Feel free to add its CO2 information to our CO2 Data Form!");}
```

Comment:
would it be possible to add a link to the our form? alternatively I can just add it in for my pr later